src/plugins/dev_tools/public/application.tsx

```diff
@@ -203,6 +203,7 @@ export function renderApp(
   });
 
   return () => {
+    chrome.docTitle.reset();
```

Comment:
I think we could add this to the management plugin (in OSS) too, so that when the management area renders/mounts, before any plugins have rendered, we always reset the doc title. Instead of in the clean up function we do this on init.

Reply:
I see what you're saying, and I think I'd like to encourage each app to clean up after itself instead. I think it's likely that we'll move apps out of Management and when that happens it will be nice to know that this behavior is encapsulated and won't be broken by the move.